apps/upload/serializers.py

```diff
@@ -23,6 +25,7 @@ def validate_file(self, file):
         if file.content_type != "image/webp":
             file = self.convert_to_webp(file)
         file.name = f'{uuid.uuid4().hex}.webp'
+        self.validate_resolution(file)
```

Comment:
Let's move validate_resolution before convertion to webp to remove redundant work if the uploaded image is invalid

Reply:
Moved validate_resolution to the beginning of the function